source/iostreams.tex

```diff
@@ -626,6 +626,7 @@
 \synopsis{Header \tcode{<ios>} synopsis}
```

Comment:
I can merge this as is, but perhaps something worth cleaning up. Let me know how you want to proceed.